2fa-api/handlers/create.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../db");
+
+const create = function (req, res) {
+  try {
+    const { email } = req.body;
+    const { password } = req.body;
+    const { phone } = req.body;
+
+    db.connection.query(
+      `INSERT INTO Users (email, password, phone) VALUES ('${email}', '${password}', '${phone}')`,
+      (error, results) => {
+        if (error) {
+          console.log(error);
+          return res.status(500).json({ error });
+        }
+
+        req.session.loggedIn = true;
+        return res.status(200).json({ data: { loggedIn: true, results } });
+      }
+    );
+  } catch (error) {
+    return res.status(500).json({ error });
```

Comment:
why is everything a 500? what do you know about error handlers?

Reply:
I'm returning a generic server error if the try block fails. Am I misusing the error handlers?

Reply:
i don't see any express errorhandler or mapping from error -> response code anywhere.

`500` indicates the server had an issue, that's not always the case.